src/facets/AccusationMultipleProposalFacet.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity >=0.5.15;
+
+import "./AccusationLibrary.sol";
+
+contract AccusationMultipleProposalFacet {
+
+    // 
+    function AccuseMultipleProposal(
+        bytes calldata _signatureGroup0,
```

Comment:
Does _signatureGroup* and _rClaims* belong to the same RCert object or they are different? In case they belong to the same RCert object, maybe we can send the serialised RCerts instead of the serialising and sending the RClaims and Signature separately. If this is possible, we can avoid some overhead bytes included by capnpro when we serialize structs separately. 

From Capn'Proto files: https://github.com/MadBase/MadNet/blob/main/consensus/objs/capn/consensus.capnp#L33
```
struct RCert {
  rClaims @0 :RClaims = .defaultRClaims;
  sigGroup @1 :Data = 0x"00";
}
```